v2.1/performance-benchmarking-with-tpc-c.md

```diff
@@ -0,0 +1,271 @@
+---
+title: Performance Benchmarking with TPC-C
+summary: Learn how to benchmark CockroachDB against TPC-C.
+toc: false
+---
+
+This page walks you through [TPC-C](http://www.tpc.org/tpcc/) performance benchmarking on CockroachDB. It measures `tpmC` (new order transactions/minute) on two TPC-C datasets:
+
+- 1,000 warehouses (for a total dataset size of 200GB) on 3 nodes
+- 10,000 warehouses (for a total dataset size of 2TB) on 30 nodes _(Coming soon)_
+
+These two points on the spectrum show how CockroachDB scales from modest-sized production workloads to larger-scale deployments.
+
+<!--This demonstrates how CockroachDB achieves high OLTP performance of over 128,000 tpmC on a TPC-C dataset over 2TB in size.-->
+
+<div id="toc"></div>
+
+## Benchmark a small cluster
+
+### Step 1. Start a 3-node cluster on Google Cloud Platform GCE
+
+Follow steps 1-6 in the [GCE tutorial to deploy a 3-node CockroachDB cluster on Google Cloud](deploy-cockroachdb-on-google-cloud-platform.html), with the following changes:
+
+- For the 3 CockroachDB nodes, use `n1-highcpu-16` VMs with [Local SSD storage](https://cloud.google.com/compute/docs/disks/local-ssd).
+    For our TPC-C benchmarking, we use `n1-highcpu-16` machines. Currently, we believe this (or higher vCPU count machines) is the best configuration for CockroachDB under high traffic scenarios. We also attach a single local SSD to each virtual machine. Local SSDs are low latency disks attached to each VM, which maximizes performance. We do not recommend using network-attached block storage. We chose this configuration because it best resembles what a bare metal deployment would look like, with machines directly connected to one physical disk each.
```

Comment:
If you want this to be a new paragraph nested under this bullet, I think you need an empty line. As-is, it's just continuing the paragraph above: 
![screen shot 2018-07-03 at 3 03 46 pm](https://user-images.githubusercontent.com/15893490/42239566-4dbf468a-7ed2-11e8-8f55-59b9b5a80fba.png)